src/HbsAsset.js

```diff
@@ -0,0 +1,20 @@
+const HTMLAsset = require("parcel-bundler/lib/assets/HTMLAsset");
+
+const handlebarsExpression = (path) => {
+  return /^{{.+?}}$/.test(path);
+};
+
+class HbsAsset extends HTMLAsset {
+  addDependency(name, options) {
+    if (!handlebarsExpression(opts.resolved))
```

Comment:
`opts` is not resolved. This should probably be `options.resolved`.

Reply:
Sorry for my typo... fixed in 7699adc